src/interactions/slashCommand.ts

```diff
@@ -368,7 +368,7 @@ export class SlashCommandPermissionsManager {
     )
     return data.map(transformSlashCommandPermissionsPayload)
   }
-
+  

```

Comment:
I think you added a bit of unnecessary bytes

Reply:
:facepalm: